operator/config/default/manager_webhook_patch.yaml

```diff
@@ -14,10 +14,6 @@ spec:
           protocol: TCP
         volumeMounts:
         - mountPath: /tmp/k8s-webhook-server/serving-certs
-          name: cert
-          readOnly: true
+          name: certs-dir
       volumes:
-      - name: cert
-        secret:
-          defaultMode: 420
-          secretName: webhook-server-cert
+      - name: certs-dir
```

Comment:
what kind of volume is this? there's nothing defined...

Reply:
the volume is declared in the manager.yaml file under the webhook deployment, not sure if it is needed to patch it again there, probably it is enough to remove it